code/modules/events/crystal_event.dm

```diff
@@ -688,7 +690,7 @@ This section is for the crystal monsters variations
 	move_resist = MOVE_FORCE_EXTREMELY_STRONG
```

Comment:
you might also want to give the boss (and/or the other crystal monsters) the TRAIT_TESLA_SHOCKIMMUNE trait, so that he doesn't zap himself or his allies to death